package.json

```diff
@@ -121,6 +121,7 @@
     "karma-webpack": "^2.0.9",
     "mocha": "^4.1.0",
     "nsp": "^3.1.0",
+    "raw-loader": "^0.5.1",
```

Comment:
I don't think this is necessary anymore. The raw loader was what we used to load the ASCII Lockie asset, but since we have a fancy SVG now, we don't need the raw loader for anything.

Reply:
I'll remove that.